model/src/cfd.rs

```diff
@@ -1866,7 +1866,7 @@ impl Dlc {
 
             (outpoint, amount)
         };
-        let (tx, sighash) = maia::close_transaction(
+        let (tx, sighash) = maia_deprecated::close_transaction(
```

Comment:
❓ Can you remind me why we want to use the buggy version unconditionally? And maybe add the explanation explicitly to the commit message if possible.

Reply:
we are not rolling out collab close on libp2p just yet, and this code has changed quite a bit on the branch due to the fact that we've been redesigning the protocol so that both parties publish.
If we end up using the same function, then we'll need to add a condition here, yes. I figured it would be easier to assess on the collab close branch (and for now the crucial bit is preserving the backwards compatibility).

I will put this in the comment here and also in the commit message